login/views.py

```diff
@@ -0,0 +1,34 @@
+import json
+from django.shortcuts import render
+from django.http import HttpResponse, HttpResponseBadRequest
+from django.views.generic import View
+from django.contrib.auth.views import LoginView
+from django.contrib.auth import authenticate
+
+
+class IndexView(LoginView):
+    template_name = '../templates/index.html'
+
+    def post(self, request, *args, **kwargs):
+        return render(request, 'index.html')
+
+
+class IndexViewPost(View):
+    # @ensure_csrf_cookie
```

Comment:
@zairiel243 remove comment. remove this comment since doesn't help at all like for documentation or something.